packages/shopify-app-session-storage-sqlite/src/sqlite.ts

```diff
@@ -116,30 +133,33 @@ export class SQLiteSessionStorage implements SessionStorage {
           state varchar(255) NOT NULL,
           isOnline integer NOT NULL,
           expires integer,
-          scope varchar(255),
+          scope varchar(1024), -- sqlite allows more than this limit without failing
```

Comment:
```suggestion
          scope varchar(1024),
```
Why would it be important to keep this comment?